website/docs/r/host_virtual_switch.html.markdown

```diff
@@ -174,3 +174,18 @@ the resource. This is set to an ID value unique to Terraform - the convention
 is a prefix, the host system ID, and the virtual switch name. An example would
 be `tf-HostVirtualSwitch:host-10:vSwitchTerraformTest`.
 
+## Importing
+
+An existing vSwitch can be [imported][docs-import] into this resource by its id.
+The convention of the id is a prefix, the host system ID, and the virtual switch
```

Comment:
One last thing on this, can ```host system ID``` be changed to ```host system [managed objectID][docs-about-morefs]```? And add:
```[docs-about-morefs]: /docs/providers/vsphere/index.html#use-of-managed-object-references-by-the-vsphere-provider```

There are several different ID types used throughout the provider, so it is just helpful to be extra clear when specifying.